code/modules/hydroponics/grown/cannabis.dm

```diff
@@ -62,20 +62,24 @@
 	plantname = "Omega Weed"
 	product = /obj/item/weapon/reagent_containers/food/snacks/grown/cannabis/ultimate
 	mutatelist = list()
-	reagents_add = list("space_drugs" = 0.3,
-						"lsd" = 0.3,
+	reagents_add = list("lsd" = 0.15,
+						"suicider" = 0.15,
+						"space_drugs" = 0.15,
 						"mercury" = 0.15,
 						"lithium" = 0.15,
 						"atropine" = 0.15,
+						"ephedrine" = 0.15,
```

Comment:
Aside from cat drugs, this is the content of omega weed as of their public release.